docs/contributing/developing.md

```diff
@@ -1,4 +1,4 @@
-(developing)=
+(developing-wagtail)=
```

Comment:
```suggestion
(developing_for_wagtail)=
```

Please use underscores for refs - alternatively skip this change completely and use `developing.md` in the index page.